packages/pn-personagiuridica-webapp/src/pages/ApiIntegration.page.tsx

```diff
@@ -32,11 +45,13 @@ const ApiIntegration: React.FC = () => {
           subTitle={t('subTitle')}
           variantSubTitle="body1"
         />
+        {!integrationApiIsEnabled && <IntegrationApiBanner isAdminWithoutGroups={isAdminWithoutGroups}/>}
         {isAdminWithoutGroups && <PublicKeys />}
-        {hasValidKey && <VirtualKeys />}
+        <VirtualKeys integrationApiIsEnabled={integrationApiIsEnabled} />
```

Comment:
Shouldn't the section be shown if I have virtual keys or my organization has an active public key? Here it is always shown.